tests/runners/bulk_update_runner.py

```diff
@@ -10,7 +10,7 @@
 from tests.runners.submission_manager import SubmissionManager
 from tests.utils import Progress
 
-MODIFIED_BSD_ACCESSION_ID = '999'
+MODIFIED_INSDC_ACCESSION_ID = '999'
```

Comment:
It would be nice to pick a valid value and verify that it's valid ;)

Reply:
This is just the delta